src/modules/chat/index.js

```diff
@@ -157,6 +159,14 @@ class ChatModule {
                     continue;
                 }
 
+                // check steam parasbility
+                const steamJoinLink = part.match(STEAM_LOBBY_JOIN_REGEX);
+                if (part.length > 0 && part.substring(0, 18) === 'steam://joinlobby/' && steamJoinLink && steamJoinLink[0]) {
```

Comment:
all of these checks except the 3rd one are unnecessary. If the regex matches, then by definition 1, 2 and 4 must be true.

Reply:
removed unnecessary checks